app/components/provider_interface/course_presentation_component.rb

```diff
@@ -0,0 +1,17 @@
+module ProviderInterface
+  class CoursePresentationComponent < ActionView::Component::Base
```

Comment:
This (and LocationPresenterComponent) needs to be aware of the possibility that a different course could be offered, eg looking at `ApplicationChoice#offered_course`

Reply:
Good catch, will make these components aware of this.